### ISSUE TITLE
Cannot provision Schematics workspaces with recent versions of Terraform

### ISSUE THREAD

**Issue by andrewferrier** [IBM-Cloud/terraform-provider-ibm#3048]
### Terraform CLI and Terraform IBM Provider Version

```
Terraform v1.0.5
on darwin_amd64
+ provider registry.terraform.io/ibm-cloud/ibm v1.30.2
```

### Affected Resource(s)

* ibm_schematics_workspace

### Terraform Configuration Files

```hcl
resource "ibm_schematics_workspace" "schematics_workspace_instance" {
  name = "schematics-test"
  description = "description"
  location = "eu-gb"
  template_type = "terraform_v1.0"
  template_git_url = "https://eu-gb.git.cloud.ibm.com/some/infrastructure"
}
```

### Debug Output

│ Error: "template_type" ("terraform_v1.0") should match regexp ^terraform_v0\.(?:11|12|13)(?:\.\d+)?$
│
│   with ibm_schematics_workspace.schematics_workspace_instance,
│   on main.tf line 22, in resource "ibm_schematics_workspace" "schematics_workspace_instance":
│   22:   template_type = "terraform_v1.0"

### Expected Behavior

The provider should support all versions of Terraform supported by IBM Cloud Schematics at all times, including 1.0 - this hardcoded limitation is overly restrictive.

### Actual Behavior

Restricts me to using old versions of Terraform.

### Steps to Reproduce

1. `terraform apply`

**Comment by nven** [comment]
Will be fixed as part of #2901

**Comment by kavya498** [comment]
Available in 1.36.0 release
Closing this issue 
Thanks
